resources/prometheus/prometheus-rules.yaml

```diff
@@ -307,8 +307,8 @@ spec:
         # GRPC
         - expr: |
             sum by (namespace, rhacs_instance_id, rhacs_org_id, rhacs_org_name, rhacs_cluster_name, rhacs_environment)
-            (rate(grpc_server_handled_total{namespace=~"rhacs-.*", job="central", grpc_type="unary", grpc_code="OK"}[10m]))
-          record: central:grpc_server_handled:OK:rate10m
+            (rate(grpc_server_handled_total{namespace=~"rhacs-.*", job="central", grpc_type="unary", grpc_code!~"DataLoss|DeadlineExceeded|Internal|Unavailable|Unknown"}[10m]))
```

Comment:
`DataLoss` is "never generated by the library": https://grpc.github.io/grpc/core/md_doc_statuscodes.html

Hence, we would only see it if Central (itself or proxying Scanner) explicitly returns it. Here's my naive check:

```
% find scanner stackrox -type f -name '*.go' | xargs grep -n DataLoss                    
%
```

If we don't explicitly return, is there any reason why introduce it? We could leave it out and assume that all codes listed capture the unavailability scenarios.

Thanks for capturing "Unknown".

Reply:
What if we decide to return it in the future? No harm imo to already include it.

Reply:
> What if we decide to return it in the future? 

Given that we don't use it today, we would have to coordinate with Sensor if we decide to use this new status in the future. We might prefer using the other statuses, the ones we are adding (excluding `DataLoss`). I don't have enough info to make that decision right now.

>  No harm imo to already include it.

Generally yes, but I think there are minor drawbacks to adding this status now. First, it's confusing because we don't use it. Why do we have it listed? Second, considering the metric can only be returned if the server explicitly decides to, someone might inadvertently return it, which will impact our availability metric. So unless an immediate reason to add, I think we should just leave it out.

Reply:
Fair enough